public_html/assets/css/nf-core-dark.css

```
@@ -498,3 +498,17 @@ hr {
   color: #999;
   background-color: #181a1b;
 }
+.breadcrumb {
+  background-color: #616161;
+  /* border-color: #616161; */
+  /* color: #ededed; */
+}
+.breadcrumb-item.active{
+  color: #e5e6e7;
+}
+.breadcrumb-item+.breadcrumb-item::before{
+  color: #e5e6e7;
+}
+#file-preview iframe{
+  background-color: #e5e6e7;
+}
```

I think maybe giving this a max-height with scroll is a good idea - I tried it on a CSV file and it was mega long, so getting to the page footer was going to take forever. Scrolling within the page feels a bit more manageable..

```suggestion
}
#file-preview .card-body {
  max-height: 30rem;
  overflow-y: auto;
}
```